Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Card.css';
+
+function CardBanner (props) {
+    return(
+        <React.Fragment>
+            <img className='ContentImg' src="https://reactjs.org/logo-og.png" alt='react logo'/>
+        </React.Fragment>
```

Comment:
I see you are familiar with Fragment :)  Useful for times like this when you don't want to litter your app with `<div>`s